cms/static_src/sass/components/_contact-details.scss

```diff
@@ -1,7 +1,14 @@
 @use 'config' as *;
 
 .contact-details {
```

Comment:
question: will this be contributed back to DS in a different PR?

Reply:
Not needed now?